docs/community.md

```diff
@@ -19,6 +19,3 @@ Feedback and questions are welcome at [support@treeverse.io](mailto:support@tree
 
 Join our [channel](https://join.slack.com/t/lakefs/shared_invite/zt-g86mkroy-186GzaxR4xOar1i1Us0bzw) to get help from our team and other users
 
-### Forum
-
-Coming soon
```

Comment:
Hi!

Thanks for your suggestion.  I'd prefer not to add a "coming soon" now and shortly afterward a link to an actual forum.  Ideally you would add a PR with a link once there is an active forum.

Thanks!

Reply:
Hello!

I think I misunderstood your response but I mean to remove the "coming soon" and not add. If you meant it should be replaced with a forum link instead let me know :)

Thanks!
@arielshaqed